### PR TITLE
improve MeanMedianImputer's user guide

### PR DESCRIPTION
**Pull request by cmcouto-silva**
I've updated the text for the MeanMedianImputer User Guide, adding a figure to showcase the distribution change.

### CODE REVIEW

File: docs/user_guide/imputation/MeanMedianImputer.rst [hunk 1]
```diff
@@ -5,100 +5,305 @@
...
+	mmi.imputer_dict_
 
-	# Separate into train and test sets
-	X_train, X_test, y_train, y_test = train_test_split(
-                                            data.drop(['Id', 'SalePrice'], axis=1),
-                                            data['SalePrice'],
-                                            test_size=0.3,
-                                            random_state=0,
-                                            )
+The `.imputer_dict_` attribute shows the learned statistics:
```
Comment (solegalli) [inline docs/user_guide/imputation/MeanMedianImputer.rst:169]:
```suggestion
The `imputer_dict_` attribute shows the learned statistics:
```

File: docs/user_guide/imputation/MeanMedianImputer.rst [hunk 2]
```diff
@@ -5,100 +5,305 @@
...
+	650       Somerst       65.000   105.26104
+	1018      Gilbert       70.375    76.00000
 
-	# fit the imputer
-	median_imputer.fit(X_train)
 
-With fit, the :class:`MeanMedianImputer()` learned the median values for the indicated
-variables and stored it in one of its attributes. We can now go ahead and impute both
-the train and the test sets.
+Imputing missing values alongside missing indicator
```
Comment (solegalli) [inline docs/user_guide/imputation/MeanMedianImputer.rst:196]:
```suggestion
Imputing missing values alongside missing indicators
```

File: docs/user_guide/imputation/MeanMedianImputer.rst [hunk 3]
```diff
@@ -5,100 +5,305 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mean or median imputation are commonly done alongside adding missing indicators.
+We can add missing indicators with :class:`AddMissingIndicator()` from feature-engine.
```
Comment (solegalli) [inline docs/user_guide/imputation/MeanMedianImputer.rst:200]:
```suggestion
We can add missing indicators with :class:`AddMissingIndicator()` from Feature-engine.
```

**Review by solegalli — COMMENTED** [review]
Thank you @cmcouto-silva! Some minor suggestions and this is good to go :)

File: docs/user_guide/imputation/MeanMedianImputer.rst [hunk 4]
```diff
@@ -5,100 +5,305 @@
...
+	1018      Gilbert       70.375    76.00000
 
-	# fit the imputer
-	median_imputer.fit(X_train)
 
-With fit, the :class:`MeanMedianImputer()` learned the median values for the indicated
-variables and stored it in one of its attributes. We can now go ahead and impute both
-the train and the test sets.
+Imputing missing values alongside missing indicators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (solegalli) [inline docs/user_guide/imputation/MeanMedianImputer.rst:197]:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment (solegalli) [reply docs/user_guide/imputation/MeanMedianImputer.rst:197]: that should fix the error in the tests

Comment (cmcouto-silva) [reply docs/user_guide/imputation/MeanMedianImputer.rst:197]: ok, trying it out!

**Pull request merged**